.github/workflows/unit-tests-bsd.yml

```diff
@@ -21,30 +21,29 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  # BUG: These tests are broken
```

Comment:
The tests broke, because the rust version in FreeBSD package manager is `1.79` which creates these warnings on compile.